configurations/off.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
```

Comment:
Hmm... I think it is descriptive enough if you only import / define the rules that you would like to enable. As no eslint rules are enabled by default, when would someone need to import this?